crates/astria-sequencer/README.md

```diff
@@ -105,7 +105,8 @@ cometbft init
 # application state
 ../../target/debug/astria-sequencer-utils \
     --genesis-app-state-file=test-genesis-app-state.json \
-    --destination-genesis-file=$HOME/.cometbft/config/genesis.json
+    --destination-genesis-file=$HOME/.cometbft/config/genesis.json \
+    --chain-id <desired_test_chain_id>
```

Comment:
Very nit: seems orthogonal to the bump but makes sense to do this here (given that your testing didn't work).